README.md

```diff
@@ -4,49 +4,230 @@ Provides resources for installing and configuring both PowerDNS authoritative an
+**
+
+The current version of the cookbook provides basic support for recursors and authoritative servers with a handful of platforms, backends and init systems supported. You can find what is supported in this table:
+
+
+| Platform | Backends         | Init Systems |
+|----------|------------------|--------------|
+| Debian   | bind, postgresql | SysVinit     |
+| CentOS   | bind, postgresql | SysVinit     |
 
 ### Platforms:
 
-* Ubuntu (14.04, 16.04)
-* Debian (8)
-* CentOS (6.8, 7.2)
+* Ubuntu (14.04)
+* CentOS (6.8)
 
 ### Chef:
 
 - Chef 12.5+
 
+### Init Systems:
+
+Only SysVinit is supported for now, Systemd is next, and along it other platforms such us Debian 8, Ubuntu 16.04 and CentOS 7.2 will be available.
+
 ### Required Cookbooks:
 
 * apt
 * yum
 
 ### Suggested Cookbooks:
 
-* mysql (for the MySQL backend)
-* sqlite (for the SQLite backend)
 * postgres (for the PostgreSQL backend)
 
 ## Usage
 
-Use the `pdns_recursor_install`, `pdns_recursor_config`, and `pdns_recursor_service` resources in your wrapper cookbooks to install, configure, and define PowerDNS recursors. Set the different properties on the resources according to your install and configuration needs. You can see a good example on this on `test/cookbooks/pdns_test/recipes_recursor_install_single.rb`
+Combine the different resources in order to install, configure and manage your PowerDNS instances. This is a list of resouces that can be used:
 
-Some properties need to set consistently accross `pdns_recursor_config` and `pdns_recursor_service`. They will be noted in their specific sections with a (C). 
+  | Resource                            | Functionality                                     |
+  |-------------------------------------|---------------------------------------------------|
+  | pdns_authoritative_install          | Installs an authoritative server                  |
+  | pdns_authoritative_config           | Configures an authoritative instance              |
+  | pdns_authoritative_service          | Manages an authoritative instance                 |
+  | pdns_authoritative_backend          | Installs authoritative backend                    |
+  | pdns_recursor_install               | Installs a recusor                                |
+  | pdns_recursor_config                | Configures a recursor instance                    |
+  | pdns_recursor_service               | Manages a a recursor instance                     | 
+
+To fully configure an authoritative server you need to add at least 3 resources to your run list, `pdns_authoritative_install`, `pdns_authoritative_config` and `pdns_authoritative_service`. If you want to install any backend than the default (bind) for the authoritative server you need to add a fourth resource: `pdns_authoritative_backend`. There is a some good usage examples on `test/cookbooks/pdns_test/recipes/`.
```

Comment:
s/run list/recipe/ and s/backend than/backend other than/ and s/There is a/There are/ and s/examples on/examples in/